brainstormforce/astra
# AST-1421 - astraToggleSetupPro is not defined JS error thrown

### Description
- JS error of not defined 

### Screenshots
- https://share.bsf.io/Wnuk8mDP

### Types of changes
- Bug fix 

### How has this been tested?
- Check in customizer and in frontend as well
- Check by altering offcanvas styles with sticky header & pro options

### Checklist:
